exchanges/retry/README.md

```diff
@@ -0,0 +1,67 @@
+# @urql/exchange-retry (Exchange factory)
+
+`@urql/exchange-retry` is an exchange for the [`urql`](../../README.md) GraphQL client that allows operations (queries, mutations, subscriptions) to be retried based on an `options` parameter.
+
+The `retryExchange` is of type `Options => Exchange`.
+
+It periodically retries requests that fail due to network errors. It accepts five optional options:
+
+- `initialDelayMs` - the minimum delay between retries, defaults to 1000
+- `maxDelayMs` - the maximum delay that can be applied to an operation, defaults to 15000
+- `randomDelay` - whether to apply a random delay to protect against thundering herd, defaults to true
+- `maxNumberAttempts` - the maximum number of attempts to retry any given operation, defaults to Infinity
+- `retryIf` - optional function to apply to errors to determine whether they should be retried
+
+The `retryExchange` will exponentially increase the delay from `minDelayMs` up to `maxDelayMs` with some random jitter added to avoid the [thundering herd problem](https://en.wikipedia.org/wiki/Thundering_herd_problem).
+
+## Quick Start Guide
+
+First install `@urql/exchange-retry` alongside `urql`:
+
+```sh
+yarn add @urql/exchange-retry
+# or
+npm install --save @urql/exchange-retry
+```
+
+You'll then need to add the `retryExchange`, that this package exposes, to your
+`urql` Client:
+
+```js
+import { createClient, dedupExchange, cacheExchange, fetchExchange } from 'urql';
+import { retryExchange } from '@urql/exchange-retry';
+
+const client = createClient({
+  url: 'http://localhost:1234/graphql',
+  exchanges: [
+    dedupExchange,
+    cacheExchange,
+    fetchExchange,
+    retryExchange(options), // Use the retryExchange factory to add a new exchange
```

Comment:
I'd define some basic options object since there's non visible in this context, imo a getting started can just be `copy-paste`.